src/StaticWebAssetsSdk/Tasks/GenerateStaticWebAssetEndpointsManifest.cs

```diff
@@ -36,38 +38,7 @@ public override bool Execute()
 
         try
         {
```

Comment:
Is there any meaningful change to this file or did you just move the code into a helper method. If so, why?

Reply:
Ah, I see now why. I don't think we should do it this way.

`WriteImportMapToHtml` should operate at a lower level, directly with the endpoints.